flight/Libraries/math/pid.c

```diff
@@ -47,7 +47,7 @@ static float deriv_gamma = 1.0;
  * @param[in] dT  The time step
  * @returns Output the computed controller value
  */
-float pid_apply(struct pid *pid, const float err, float dT)
+float pid_apply(struct pid *pid, float err, float dT)
```

Comment:
Starting a new review pass.

This function's signature and the next one should not change.